pkgs/top-level/all-packages.nix

```diff
@@ -33694,4 +33694,6 @@ with pkgs;
   };
 
   zthrottle = callPackage ../tools/misc/zthrottle { };
+
+  notion-app-enhanced = callPackage ../applications/office/notion-app-enhanced { };
```

Comment:
Please sort the entry into a fitting place.

Reply:
@SuperSandro2000 
Fixed.